input_files/strtx_III_dsp/dsp_hb_settings.txt

```diff
@@ -129,8 +134,8 @@ core_utilization=0.90
 
 
 # EDI settings:
-power_ring_width=3
-power_ring_spacing=3
+power_ring_width=1.8

```

Comment:
If these came from some recommended practice, mention where they came from. If you know what EDI is, say what it stands for.

Reply:
I can't remember if I got it from somewhere or if I did it myself. I'll keep looking through some guides but the reason I have those values is s.t the power rings will be outside of the core area, which seems to be general practice and the previously values always caused overlap. My calculations 1.8*2 (ring for VDD and VSS) + 1.8*3 (spacing between IO ring and first ring, the rings and last ring and core)  = 9 um (1 um of extra space)